core/services/src/main/java/org/exoplatform/services/cms/listeners/NewGroupListener.java

```diff
@@ -94,10 +95,16 @@ private void removeGroup(String groupId) throws RepositoryException
       String systemWorkspace = manageableRepository.getConfiguration().getDefaultWorkspaceName();
       Session session = manageableRepository.getSystemSession(systemWorkspace);
       Node groupNode = (Node)session.getItem(groupsPath_ + groupId);
-      groupNode.remove();
-      session.save();
-      session.logout();
-   }
+      try {
+        Utils.removeDeadSymlinks(groupNode);

```

Comment:
```suggestion
        Utils.removeDeadSymlinks(groupNode, false);
```
Removed symlinks should be added to trash but remove definitively